README.md

```diff
@@ -265,6 +265,31 @@ This will now rotate it's children over and over again, for example our logo:
   </a>
 </div>
 
+### Refs to DOM nodes
+
+Passing `ref` to styled component will give a ref to the `StyledComponent`
+wrapper, not to DOM node. So it's not possible to call DOM methods, like focus
+on that wrapper. To get a `ref` to wrapped DOM node, pass `innerRef` prop. Note
+that this only supports callback-based refs, not string-based ones.
```

Comment:
Make this note look like all our other notes like so:

```Markdown
…pass `innerRef` prop.


> **Note:** `innerRef` only supports callback refs (i.e. `ref={comp => this.bla = comp}`), string refs (i.e. `ref="bla"`) won't work. Since string based refs will be deprecated in the future anyway, don't worry about it too much and just use the callback pattern.
```